gui/options.cpp

```diff
@@ -2613,10 +2615,12 @@ void GlobalOptionsDialog::apply() {
 	bool isRebuildNeeded = false;
 
 	Common::U32String savePath(_savePath->getLabel());
-	if (!savePath.empty() && (savePath != _("Default")))
+	if (!savePath.empty() && (savePath != _("Default")) && (!savePath.contains(_(" (temporary)"))))
```

Comment:
Instead of doing this, I'd recommend introducing another method to ConfMan, like `ConfMan.isKeyTemporary()`.